test/api/services/EnrolmentsAuthServiceSpec.scala

```diff
@@ -141,10 +142,10 @@ class EnrolmentsAuthServiceSpec extends ServiceSpec with MockAppConfig {
     "the user is an authorised individual" should {
       "return the 'Individual' user type in the user details" in new Test {
 
-        MockedAppConfig.confidenceLevelCheckEnabled.returns(ConfidenceLevelConfig(definitionEnabled = true, authValidationEnabled = false))
+        MockAppConfig.confidenceLevelCheckEnabled.returns(ConfidenceLevelConfig(definitionEnabled = true, authValidationEnabled = false))
 
-        val retrievalsResult = new ~(Some(AffinityGroup.Individual), Enrolments(Set.empty))
-        val expected         = Right(UserDetails("", "Individual", None))
```

Comment:
Please add type annotation for val expected here and in other places in the file.